### DISCUSSION TITLE
Support Subtitle Files

### DISCUSSION THREAD

**Discussion by merlinbest5** [biigle/core#470] [Ideas]
We've started collecting ROV video with a Biigle-ready version right off the ship. Our ROV tech has asked me to suggest subtitle file support for data overlays on video. This would allow them to capture one "clean" version and add a separate subtitle file vs. burning it into the video, and allow for more flexibility in the placement of the information overlaid on the screen when annotating. I haven't yet added video metadata to a project, so I'm not sure if that solves my problem, but we prefer to have the positional data etc. in view :)

**Comment by mzur** [comment]
That's a great suggestion I haven't thought of before. I only know videos with "burned-in" overlays. Using subtitle files could be much cleaner. A subtitle file could be uploaded like a metadata file for a video.

I've created an issue for this and added it to our roadmap: https://github.com/biigle/core/issues/472

It would be great if you could provide an example video+subtitle file for the implementation.
